music21/abcFormat/translate.py

```diff
@@ -276,6 +277,13 @@ def parseTokens(mh, dst, p, useMeasures):
             #ql += t.quarterLength
 
         elif isinstance(t, abcFormat.ABCNote):
+            # add the attached chord symbol
+            if len(t.chordSymbols) == 1:
+                cs_ascii = t.chordSymbols[0]
+                assert cs_ascii[0] == cs_ascii[-1] == '"'
```

Comment:
We don't do bare asserts in music21 -- can you do this as a "if ...: raise Music21Exception('...detailed information here...')" but in general with handcoded formats like ABC, if there's an error, we use environLocal.warn("information...") and continue to the next token -- it's not in general worth crashing parsing for an incorrectly formatted symbol in ABC/Humdrum, etc.  (machine coded formats such as MusicXML and MIDI are different, since it's up to the creator to produce valid code).